src/main/java/wolox/gmjavatraining/repositories/BookRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.gmjavatraining.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.gmjavatraining.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+    Book findBookByAuthor(String author);
```

Comment:
You should use Optional here, you can read more about Java Optionals [here](https://www.baeldung.com/java-optional)